core/modes/modes.py

```diff
@@ -5,10 +5,8 @@
 modes = {
     "admin": "enable extra administration commands terminal (docker, etc)",
     "debug": "a way to force debugger commands to be loaded",
-    "gdb": "a way to force gdb commands to be loaded",
```

Comment:
I think we forgot to remove the voice commands activating this mode

https://github.com/knausj85/knausj_talon/blob/99869cbaebe68ec43f4ed992e0317deb9bd01070/core/modes/language_modes.talon#L3-L4